changelogs/fragments/693-big-revamp-on-xfconf-adding-array-values.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- Adds arrays support for xfconf module. (https://github.com/ansible/ansible/issues/46308)
```

```suggestion
- xfconf - add arrays support for xfconf module (https://github.com/ansible/ansible/issues/46308).
```
(See the examples in `devel`. They've been standardized recently)

Is this ready for merge after fixing the fragment or you're gonna write tests?